app/controllers/assignments_controller.rb

```diff
@@ -611,9 +611,6 @@ def process_assignment_form(assignment)
       # Is the instructor forming groups?
       if assignment_params[:assignment_properties_attributes][:student_form_groups] == '0'
         assignment.invalid_override = true
-        # Increase group_max so that create_all_groups button is not displayed
-        # in the groups view.
-        assignment.group_max = 2
```

Comment:
Can you explain the purpose of this change?

Reply:
If groups are enabled but students can form their own group is disabled, the group_max value cannot be changed from 2. This means that an instructor who wants to manage their own groups, but wants to have groups that are larger than 1 student each, will have to manually validate every group they make on the groups page (because MarkUs will think that the group size is invalid).

This was here originally because we didn't want instructors to be able to click the "Create all groups" button if they said that students could form groups. However, this button is only visible if they set the max group size to 1. In that case, it would actually be very convenient for instructors to be able to create all remaining groups for the rest of the students with a click of a button.

Reply:
Great, thanks!